linode/manager
# M3-5752: Fixed Tags Panel causing extra PUT requests

## Description

- Fixes Tags panel causing a PUT request on mount
  - Caused by #8089
- Flex Tags panel not allowing you to create a new tag

## How to test

- Text Tags panel functionality against production
- Check to make sure the extra PUT request on mount has stopped occurring
